### ISSUE TITLE
A bug in sss_meas_data.cpp?

### ISSUE THREAD

**Issue by xyp8023** [nilsbore/auvlib#61]
Hi Nils,
In this line: 
https://github.com/nilsbore/auvlib/blob/44e55eb1b51786e5673bf7d3d00c51cc21e4f892/src/bathy_maps/src/sss_meas_data.cpp#L177
I assume if ind==waterfall_width/2-1, it means the furthest point from the origin, and if we are going to fill in the hits_X (with width waterfall_width) for the left side, the corresponding column is 0, right? Not col = waterfall_width/2 + ind = waterfall_width-1 as written in the code.

So maybe it should be changed to:
 `if (ind == -1) {
            continue;
        }
        else if (!is_left) {
            col = waterfall_width/2 + ind;
        }
        else {
            col = waterfall_width/2 - 1 - ind;
        }`

**Comment by xyp8023** [comment]
Wait, in the waterfall image, say sss_waterfall_hits_X with shape (928,512), the left side of waterfall sss_waterfall_hits_X[:,:256] is corresponding the port side or stbd side of the auv? If we assume the auv is moving from the top to the bottom, then the left side of waterfall image is the stbd side of the auv, then in this case, I think I used the wrong origin to calculate the the slant range as I described in the email. Is it merely I misused the origin_port and origin_stbd instead of a bug?
